katharsis-core/src/main/java/io/katharsis/legacy/queryParams/include/Inclusion.java

```diff
@@ -29,6 +29,15 @@ public String getPath() {
     }
 
     @Override
+    public int compareTo(Inclusion o) {
```

Comment:
Java 8 has some really nice ways to build comparators functionally and even putting nulls first/last. At a minimum, could we have a comment on how you _think_ this sort should work?